### PR TITLE
Update Readme.txt

### PR DESCRIPTION
**Pull request by hExPY**
Adding an explanation of how to run the script in the background; Adding an explanation of how to kill the process when it has been executed in the background

### CODE REVIEW

**Comment by ShogunMan** [comment]
Is this is just for linux installations?

**Comment by hExPY** [comment]
@ShogunMan oh yes. I'll add it to the Doc.

**Comment by CyberPunkMetalHead** [comment]
@hExPY let me know when you've added that to the file and I will merge your request

**Comment by CyberPunkMetalHead** [comment]
My bad, you already did!

File: Readme.txt [hunk 1]
```diff
@@ -13,3 +13,8 @@ Requirements: python-binance .7.9
 
 For a step-by-step guide on how to implement and configure this bot please see the guide at: 
 https://www.cryptomaton.org/2021/05/08/how-to-code-a-binance-trading-bot-that-detects-the-most-volatile-coins-on-binance/
+
+In order to run this script in the background without demonizing, you can execute it in the following way (Linux only):
```
[the remark this answers is not in the document]
Comment (hExPY) [reply Readme.txt:17]: You are absolutely right. Screen would work as well. I wanted to provide a solution using as less as possible dependencies.

**Pull request merged**